README.md

```diff
@@ -513,6 +513,49 @@ class UserDetailsController extends BaseHttpController {
 }
 ```
 
+### Decorators
+There are helpers for AuthProvider these are wrapped into decorators these are applied to action methods.
```

Comment:
this English seems a bit shady, I am not a native speaker myself so can't really help with correct phrasing